libs/Helpers.py

```diff
@@ -47,7 +47,9 @@ def gte(self, version, target):
     def install_rhoam_addon(self, cluster_name):
         ocm_client = OpenshiftClusterManager()
         ocm_client.cluster_name = cluster_name
-        ocm_client.install_rhoam_addon(exit_on_failure=False)
+        result = ocm_client.install_rhoam_addon(exit_on_failure=False)
+        if not result:
+            raise FatalError("Something got wrong while installing RHOAM. Check the logs")
```

Comment:
Please, replace FatalError by built-in fail

Reply:
done